init.lua

```diff
@@ -5,6 +5,8 @@ require('cudnn.ffi')
 local C = cudnn.C
 local ffi = require 'ffi'
 
+local thc = ffi.load("THC")
```

Comment:
the symbols of THC will be loaded twice into the process. I am not fully sure around the semantics of what will happen, it'll be dependent on the flags given to dlopen. For example, there might be two separate THCState structs living in the same project, initialized differently, depending on the default dlopen flags